AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.19.0.3. Set the 'ServerName' directive globally to suppress this message
[Mon Mar 21 16:00:44.580151 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Mon Mar 21 16:00:44.580203 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Mon Mar 21 16:01:25.222996 2022] [autoindex:error] [pid 17] [client 172.19.0.1:43312] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:19:55.180240 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.3. Set the 'ServerName' directive globally to suppress this message
[Mon Mar 21 16:21:30.914347 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Mon Mar 21 16:21:30.914393 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Mon Mar 21 16:22:13.988910 2022] [autoindex:error] [pid 17] [client 172.20.0.1:58730] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:22:14.766257 2022] [autoindex:error] [pid 17] [client 172.20.0.1:58730] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:22:15.234612 2022] [autoindex:error] [pid 17] [client 172.20.0.1:58730] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:22:15.394006 2022] [autoindex:error] [pid 17] [client 172.20.0.1:58730] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:22:30.673601 2022] [php:notice] [pid 18] [client 172.20.0.1:58732] 2022-03-21T16:22:30+00:00 [critical] Uncaught Exception: Environment variable not found: "DATABASE_URL".
[Mon Mar 21 16:23:18.000047 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.3. Set the 'ServerName' directive globally to suppress this message
[Mon Mar 21 16:25:38.262088 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Mon Mar 21 16:25:38.262134 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Mon Mar 21 16:27:22.269578 2022] [autoindex:error] [pid 19] [client 172.21.0.1:46496] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:27:28.320187 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.3. Set the 'ServerName' directive globally to suppress this message
[Mon Mar 21 16:27:43.189867 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Mon Mar 21 16:27:43.189937 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Mon Mar 21 16:28:00.942238 2022] [autoindex:error] [pid 17] [client 172.21.0.1:46502] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:29:04.091658 2022] [autoindex:error] [pid 19] [client 172.21.0.1:46506] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:29:37.791441 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.3. Set the 'ServerName' directive globally to suppress this message
[Mon Mar 21 16:31:01.349701 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Mon Mar 21 16:31:01.349764 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Mon Mar 21 16:31:34.579951 2022] [autoindex:error] [pid 18] [client 172.21.0.1:46508] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:31:35.331524 2022] [autoindex:error] [pid 18] [client 172.21.0.1:46508] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:31:59.097549 2022] [autoindex:error] [pid 22] [client 172.21.0.1:46520] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:32:12.312199 2022] [autoindex:error] [pid 64] [client 172.21.0.1:46518] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
[Mon Mar 21 16:52:54.932294 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.19.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 08:53:14.841692 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 08:53:14.841782 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 10:17:41.269703 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 10:18:07.406862 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 10:18:07.406921 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 10:24:01.013975 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 10:24:02.765321 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 10:24:02.765382 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 10:25:21.077359 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 10:25:29.173000 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 10:25:29.173060 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 11:39:09.240093 2022] [php:notice] [pid 17] [client 172.20.0.1:42732] 2022-03-22T11:39:09+00:00 [critical] Uncaught Error: Unclosed '{' on line 24 does not match ')'
[Tue Mar 22 12:42:35.410028 2022] [core:notice] [pid 1] AH00051: child pid 21 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Tue Mar 22 13:23:59.763379 2022] [php:notice] [pid 103] [client 172.20.0.1:42954] 2022-03-22T13:23:59+00:00 [critical] Uncaught Exception: The file "/var/www/html/config/packages/twig.yaml" does not contain valid YAML: Unable to parse at line 3 (near "    form_themes: ['bootstrap_5_layout.html.twig']") in /var/www/html/config/packages/twig.yaml (which is being imported from "/var/www/html/src/Kernel.php").
[Tue Mar 22 13:57:15.901659 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.4. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 13:57:40.488038 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 13:57:40.488093 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 14:17:43.834137 2022] [core:notice] [pid 1] AH00051: child pid 20 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Tue Mar 22 14:18:21.319924 2022] [php:notice] [pid 24] [client 172.21.0.1:52770] 2022-03-22T14:18:21+00:00 [critical] Uncaught Error: syntax error, unexpected token "}"
[Tue Mar 22 16:02:25.848266 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 19:22:16.615575 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 19:22:16.615628 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Tue Mar 22 19:24:07.814215 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.442158 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.442567 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.468287 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.696039 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.696120 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.759289 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.759463 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.759488 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.823360 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.823376 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.885291 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.885320 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.947320 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.947328 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.972818 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:08.972913 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.009410 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.009692 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.035665 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.035739 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.072020 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.072075 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.098906 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.098919 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.098942 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.134146 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.134742 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.161274 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.161356 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.175604 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.176288 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.224735 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.225063 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.239517 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.239830 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.267535 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.267780 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.301206 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.301243 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.331141 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.331212 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.331499 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.363688 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.363717 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.372789 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.372886 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.373116 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.426526 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.426537 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.426595 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.426686 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.426881 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.488821 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.488869 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.488922 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.488964 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.488993 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.530584 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.530601 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.530629 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.530642 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.530679 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.593285 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.593360 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.593415 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.593743 2022] [php:notice] [pid 19] [client 172.21.0.1:47596] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.593818 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.634965 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.634972 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.635026 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.635064 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.635088 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.635133 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.697081 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.697154 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.697181 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.697221 2022] [php:notice] [pid 17] [client 172.21.0.1:47590] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.697258 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.760157 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.760220 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.760221 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.760288 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.822892 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.822960 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.823034 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.823051 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.823082 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.865506 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.865664 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.865682 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.865742 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.865804 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.929288 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.929481 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.929521 2022] [php:notice] [pid 20] [client 172.21.0.1:47598] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.929596 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.929694 2022] [php:notice] [pid 18] [client 172.21.0.1:47594] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.994106 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.994131 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:09.994169 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.056563 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.056607 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.056895 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.134877 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.134950 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.134970 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.197261 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.197335 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.197645 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.259910 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.259988 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.260016 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.302200 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.302284 2022] [php:notice] [pid 21] [client 172.21.0.1:47600] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.302330 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.364654 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.364711 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.406907 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.407115 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.472105 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.472176 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.536011 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.536051 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.599340 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.599683 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.641602 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.641609 2022] [php:notice] [pid 22] [client 172.21.0.1:47602] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:24:10.683581 2022] [php:notice] [pid 23] [client 172.21.0.1:47604] Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(, referer: http://localhost/faktura/
[Tue Mar 22 19:42:37.866004 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.21.0.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 22 19:42:39.825332 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/8.0.10 configured -- resuming normal operations
[Tue Mar 22 19:42:39.825380 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
